### ISSUE TITLE
PIA Container broken

### ISSUE THREAD

**Issue by jgriss** [BioContainers/containers#301]
The PIA container v1.3.8_cv1 is broken: The entrypoint is set to "runpia.sh" but "runpia.sh" is not executable.

Since, for some reason, runpia.sh belongs to root even if the COPY command is added after "USER biodocker", I move the "chmod" command to before changing the user.

The patch for the Dockerfile looks as follows:
```
--- a/pia/1.3.8/Dockerfile
+++ b/pia/1.3.8/Dockerfile
@@ -24,6 +24,10 @@ ENV ZIP_FILE=pia-1.3.8.zip \
   PIA_PATH=/home/biodocker/pia \
   PIA_JAR=/home/biodocker/pia/pia-1.3.8.jar
 
+COPY ./runpia.sh $USER_HOME/bin/
+
+RUN chmod +x $USER_HOME/bin/runpia.sh
+
 USER biodocker
 
 RUN set -x \
@@ -31,8 +35,6 @@ RUN set -x \
   && unzip $USER_HOME/$ZIP_FILE -d $PIA_PATH \
   && rm $USER_HOME/$ZIP_FILE
 
-COPY ./runpia.sh $USER_HOME/bin/
-
 WORKDIR /data/
 
 ENTRYPOINT ["runpia.sh"]
```

**Comment by julianu** [comment]
Right, created a PR #302

**Comment by Raynooo** [comment]
Oops sorry I changed it already without using the PR

**Comment by Raynooo** [comment]
I'm looking at the other two versions to see if they need to be modified too before closing the issue

**Comment by Raynooo** [comment]
Version 1.3.0 had the same problem (now fixed) and 1.1.0 has another kind of issue (unrelated).

**Comment by jgriss** [comment]
Thanks for taking care of this so quickly!
